processor/metricstransformprocessor/metrics_transform_processor.go

```diff
@@ -108,6 +108,8 @@ func (mtp *metricsTransformProcessor) update(metric *metricspb.Metric, transform
 		// update label
 		if op.Action == UpdateLabel {
 			mtp.updateLabelOp(metric, op)
+		} else if op.Action == ToggleDataType {
```

Comment:
switch on the action?

Reply:
Will be changed significantly in this PR: https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/417

Just making minimal change here for now.